EXAMPLES.md

```diff
@@ -197,3 +197,121 @@ class AcsRespondController
 That is more than a few lines of code, but most of it is just logging. The SAML
 bundle does not check the InResponseTo of the assertion, so that too is included
 in this example.
+
+## SAML Authentications
+As of version 5 a Symfony Security Authentication SamlAuthenticator was added to the bundle. The simplied Authentication
```

Comment:
simplified ?

Reply:
I rewrote this portion, making it less ambiguous.